diff -p -up ./src/Makefile-sushi.am.orig ./src/Makefile-sushi.am
--- ./src/Makefile-sushi.am.orig	2012-03-05 18:48:03.489304283 +0000
+++ ./src/Makefile-sushi.am	2012-03-05 18:48:27.331306366 +0000
@@ -56,7 +56,8 @@ sushi-enum-types.c: stamp-sushi-enum-typ
 lib_LTLIBRARIES += libsushi-1.0.la
 
 libsushi_1_0_la_LIBADD = \
-    $(SUSHI_LIBS)
+    $(SUSHI_LIBS) \
+    -lfreetype
 
 libsushi_1_0_la_CPPFLAGS = \
     $(sushi_cflags)
